docs/examples/pytorch/resnet50/main.py

```diff
@@ -187,6 +186,8 @@ def main():
                                              init_method='env://')
         args.world_size = torch.distributed.get_world_size()
 
+    args.total_batch_size = args.world_size * args.batch_size
```

Comment:
Adding stuff to `args` that are not there seems a bit weird, but this script already does it with world_size.